apis/config/v1beta1/configuration_types.go

```diff
@@ -472,3 +476,17 @@ type FairSharing struct {
 	// The default strategy is ["LessThanOrEqualToFinalShare", "LessThanInitialShare"].
 	PreemptionStrategies []PreemptionStrategy `json:"preemptionStrategies,omitempty"`
 }
+
+type AdmissionFairSharing struct {
+	// usageHalfLifeTime indicates the time after which the current usage will decay by a half
+	// If set to 0, usage will be reset to 0.
+	UsageHalfLifeTime metav1.Duration `json:"usageHalfLifeTime,omitempty"`
+
+	// usageSamplingInterval indicates how often Kueue updates consumedResources in FairSharingStatus
+	UsageSamplingInterval metav1.Duration `json:"usageSamplingInterval,omitempty"`
```

Comment:
What is the default, or 0? If 0 is the default, does it just run in a hot loop?

Reply:
The default is 5mins, added a comment